packages/vaadin-avatar/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@vaadin/vaadin-avatar",
+  "version": "22.0.0-alpha5",
+  "description": "vaadin-avatar",
+  "main": "vaadin-avatar-group.js",
+  "module": "vaadin-avatar-group.js",
+  "repository": "vaadin/vaadin-avatar",
+  "keywords": [
+    "Vaadin",
+    "vaadin-avatar",
+    "web-components",
+    "web-component",
+    "polymer"
+  ],
+  "author": "Vaadin Ltd",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/vaadin/vaadin-avatar/issues"
+  },
+  "homepage": "https://vaadin.com/components",
+  "files": [
+    "vaadin-*.d.ts",
+    "vaadin-*.js",
+    "src",
+    "theme"
+  ],
+  "dependencies": {
+    "@vaadin/avatar": "^22.0.0-alpha5",
+    "@vaadin/avatar-group": "^22.0.0-alpha5",
+    "@polymer/iron-a11y-announcer": "^3.0.0",
+    "@polymer/iron-resizable-behavior": "^3.0.0",
+    "@polymer/polymer": "^3.0.0",
+    "@vaadin/vaadin-element-mixin": "^22.0.0-alpha5",
+    "@vaadin/vaadin-item": "^22.0.0-alpha5",
+    "@vaadin/vaadin-list-box": "^22.0.0-alpha5",
+    "@vaadin/vaadin-lumo-styles": "^22.0.0-alpha5",
+    "@vaadin/vaadin-material-styles": "^22.0.0-alpha5",
+    "@vaadin/vaadin-overlay": "^22.0.0-alpha5",
+    "@vaadin/vaadin-themable-mixin": "^22.0.0-alpha5"
```

Comment:
All of those dependencies can be removed.

Reply:
I just realized I also forgot to add the new packages as dependencies 😅